package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "start": "NODE_ENV=development gulp start:dev",
+    "start:dev": "ng serve --project=ng-zorro-antd-doc --configuration=development",
```

Comment:
Since `defaultConfiguration` is added, you don't need to add this script.

Reply:
> Since `defaultConfiguration` is added, you don't need to add this script.

I have already removed it as suggested.